src/drone_controller/calculation_layer/thrust_calculator.py

```diff
@@ -0,0 +1,16 @@
+# pylint: disable=too-few-public-methods
+from src.drone_controller.input_layer.drone_state import DroneState
+
+
+class ThrustCalculatorQuadroCopter:
+    """
+    Calculator for rotor thrusts.
+    """
+
+    def calc(self, drone_state: DroneState, expected_state: DroneState):
+        """
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Multi-line docstring summary should start at the first line](https://app.codacy.com/gh/treeandsea/DroneController/pullRequest?prid=5205179)